public/pages/AnomalyCharts/containers/FeatureBreakDown.tsx

```diff
@@ -64,7 +64,7 @@ export const FeatureBreakDown = React.memo((props: FeatureBreakDownProps) => {
       props.detector.enabledTime &&
       props.isFeatureDataMissing ? (
         <EuiCallOut
-          title={`Show missing data only after last enable time: ${moment(
+          title={`Show missing data only after last enabled time: ${moment(
```

Comment:
Wording seems kind of confusing here, maybe something like 'Missing data is only shown since the detector was last enabled on <enabled-time>', or 'There is missing data since the detector was last enabled on <enabled-time>'?

Reply:
Or even 'There is missing data since the detector was last enabled on (last enabled time)'

Reply:
Thanks for the idea. Maybe use `Missing data is only shown since last enabled time: <time-string>`?

Reply:
just want to keep wording concise.

Reply:
> Thanks for the idea. Maybe use `Missing data is only shown since last enabled time: <time-string>`?

Sounds good to me, agreed to keep it short